packages/bundler-plugin-core/src/debug-id.ts

```diff
@@ -0,0 +1,149 @@
+import * as fs from "fs";
+import MagicString from "magic-string";
+import * as path from "path";
+import * as util from "util";
+import { Logger } from "./sentry/logger";
+import { stringToUUID } from "./utils";
+
+// TODO: Find a more elaborate process to generate this. (Maybe with type checking and built-in minification)
+const DEBUG_ID_INJECTOR_SNIPPET =
+  ';!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="__SENTRY_DEBUG_ID__",e._sentryDebugIdIdentifier="sdbid-__SENTRY_DEBUG_ID__")}catch(e){}}();';
```

Comment:
We can build this whenever we need to change it - same approach as we use here: https://github.com/getsentry/sentry/pull/46780. I don't think we need to automate this, we can just manually run terser.

So everytime this snippet changes, we rebuild the minified JS. We can have this in follow up PR.